extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -472,17 +473,17 @@ private function processInTrackBlogPost() {
 	private function processInTrackBlogComment() {
 		// BADGE_BLOGCOMMENT
 		// is defined check if required because blogs are not enabled everywhere
-		if ( defined( 'NS_BLOG_ARTICLE_TALK' ) && $this->mTitle->getNamespace() == NS_BLOG_ARTICLE_TALK ) {
+		if( defined( 'NS_BLOG_ARTICLE_TALK' ) && $this->mTitle->getNamespace() == NS_BLOG_ARTICLE_TALK ) {
 			// handle only article/comment creating ( not editing )
-			if ( $this->mStatus->value['new'] == true ) {
+			if( $this->mStatus->value[ 'new' ] == true ) {
 				$blogPostTitle = Title::newFromText( $this->mTitle->getBaseText(), NS_BLOG_ARTICLE );
-				if ( $blogPostTitle ) {
+				if( $blogPostTitle ) {
 					$blogPostArticle = new Article( $blogPostTitle );
-					if ( empty( $this->mCounters[BADGE_BLOGCOMMENT] ) || !in_array( $blogPostArticle->getID(), $this->mCounters[BADGE_BLOGCOMMENT] ) ) {
-						if ( empty( $this->mCounters[BADGE_BLOGCOMMENT] ) ) {
-							$this->mCounters[BADGE_BLOGCOMMENT] = array();
+					if( empty( $this->mCounters[ BADGE_BLOGCOMMENT ] ) || !in_array( $blogPostArticle->getID(), $this->mCounters[ BADGE_BLOGCOMMENT ] ) ) {
+						if( empty( $this->mCounters[ BADGE_BLOGCOMMENT ] ) ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS134)